tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -108,67 +121,231 @@ if (tensorflow_ENABLE_JEMALLOC_SUPPORT)
   add_definitions(-DTENSORFLOW_USE_JEMALLOC -DJEMALLOC_EXPORT=)
 endif()
 
-# External dependencies
-include(zlib)
-include(gif)
-include(png)
-include(jpeg)
-include(lmdb)
-include(eigen)
-include(gemmlowp)
-include(jsoncpp)
-include(farmhash)
-include(fft2d)
-include(highwayhash)
-include(protobuf)
+if(tensorflow_USE_SYSTEM_ZLIB)
+
```

Comment:
please remove these extra blank lines everywhere to conform with the current code.